app/styles/tailwind.css

```
@@ -1463,87 +1203,61 @@ html:has(div[data-theme='orange']) {
-/* Chrome, Safari and Opera */
-
-.no-scrollbar::-webkit-scrollbar {
-    display: none;
-  }
-
-.no-scrollbar {
-    -ms-overflow-style: none; /* IE and Edge */
-    scrollbar-width: none; /* Firefox */
-  }
-
+@import 'utilities/scrollbar';
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)